src/nl/hannahsten/texifyidea/util/PandocUtil.kt

```diff
@@ -0,0 +1,72 @@
+package nl.hannahsten.texifyidea.util
+
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.execution.process.ProcessNotCreatedException
+import java.io.File
+import java.io.IOException
+import java.util.concurrent.TimeUnit
+
+object PandocUtil {
+
+    val isPandocInPath: Boolean by lazy {
+        "pandoc -v".runCommandWithExitCode().second == 0
+    }
+
+    fun translateHtml(htmlIn: String, isStandalone: Boolean = false): Pair<String?, String>? {
```

Comment:
In particular, it translates html from what to what?

Reply:
Latex?